Sources/Resources/Sentry.modulemap

```diff
@@ -3,4 +3,24 @@ framework module Sentry {
 
     export *
     module * { export * }
+    
+    explicit module HybridSDK {
```

Comment:
> I think we can rename it to _HybridSDK, its a hack way to prevent autocomplete to suggest it.

Sounds reasonable.

Reply:
Ok, i just tried this in one of the sample apps. Adding the headers through the `explicit model` makes them exposed, even if just importing Sentry, independent from the module name. Assuming we want to avoid this, this approach is not working either. Do you have another suggestion how to address this?

<img width="1194" alt="Bildschirmfoto 2024-10-21 um 14 15 59" src="https://github.com/user-attachments/assets/48eb3644-6be4-44e0-85c2-34c8f5df9de9">

Reply:
It's because the headers for the hybrid SDK moved from `project` to `private` visibility, so they can be exported by the explicit module. But as i understand this, this also exposes them in `Sentry`, making the `explicit module` redundant. I'm sure i'm missing something here.

Reply:
Turns out this was just an issue in the sample apps. When using the created framework in Flutter, the explicit header has to be imported.